projects/storefrontapp-e2e-cypress/cypress/integration/regression/my-account/consent-management.e2e-spec.ts

```diff
@@ -0,0 +1,42 @@
+import { viewportContext } from '../../../helpers/viewport-context';
+import {
+  consentManagementTest,
+  verifyAsAnonymous,
+} from '../../../helpers/consent-management';
+import * as login from '../../../helpers/login';
+
+viewportContext(['mobile', 'desktop'], () => {
+  describe('Consent Management Page', () => {
+    before(() => {
+      cy.window().then((win) => win.sessionStorage.clear());
+    });
+
+    describe('consent management test for anonymous user', () => {
+      verifyAsAnonymous();
+    });
+
+    describe('consent management test for logged in user', () => {
+      before(() => {
+        cy.requireLoggedIn();
+        cy.reload();
+        cy.visit('/');
+        cy.selectUserMenuOption({
```

Comment:
Please take a look at `projects/storefrontapp-e2e-cypress/cypress/helpers/notification.ts` I am using in that place functions for navigation with menu option additionally waiting for page. I think it is more safety as we are sure that navigation went well and we can proceed next actions.

Reply:
Same as my other comment, I didn't fix any of the tests outside of stock-notification e2es because it is out of scope. These improvements can be done in a new ticket.